packages/conventional-commit-lint/src/utils.ts

```diff
@@ -182,12 +185,16 @@ export async function scanPullRequest(
   if (lintError) {
     let summary =
       'Some of your commit messages failed linting.\n\nVisit [conventionalcommits.org](https://conventionalcommits.org) to learn our conventions.\n\n';
+    if (commitMessageDiffer === true) {
+      summary +=
+        'The PR title and the commit message differ, and there is only one commit.\n';
```

Comment:
This message tells you about the current situation, but doesn't tell you what you should do. Can we tell users what to do next?

Reply:
Thanks @tmatsuo! - If I understand the current code correct, this will only occur when the commits.length === 1 AND the commit message initially fails lint, so the subsequent "Run `git commit --amend` and edit your message to match Conventional Commit guidelines." will also be added to the summary.  Let me know if I've misunderstood, or there is a different action we would like to communicate to users.

My goal was to limit the change in behavior, the other idea was to add a new configuration parameter alwaysUsePrTitle = T/F.  Useful for repos which only use merge commits with the PR title, and so don't worry about single commit messages.

Reply:
Here is the intention of the current code.

Background: This bot is to enforce all the commit messages conforms to the conventional commit spec. When someone opens a PR on the Github UI for a simple change, people tend to choose a PR title incompatible with conventional commit. In that case, they often fix the PR title to please this bot.

However, the bot still complains because Github will use the commit message (not the PR title) when you merge this PR without squashing.

So the bot adds this message in the issue commment.

```
you can use Github's automerge feature with squashing, or use `automerge` label.
```

If you use Github's automerge feature with squashing, it will pick the PR title for the squashed merge commit.
If you add `automerge` label (which will trigger one of our other bots "merge-on-green"), the final commit message will be picked from the PR title.

So this bot will stop complaining for those cases when the PR title conforms to conventional commit.

Reply:
Based on feedback, retooling for https://github.com/googleapis/repo-automation-bots/issues/4769